includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -2453,6 +2459,44 @@ private function remove_disallowed_descendants( DOMElement $node, $allowed_desce
 		}
 	}
 
+	/**
+	 * Loop through node's siblings and remove them.
+	 *
+	 * @param DOMElement $node      Node.
+	 * @param string     $spec_name Spec name.
+	 */
+	private function remove_disallowed_siblings( DOMElement $node, $spec_name ) {
```

Comment:
We can't use `previousElementSibling` nor `nextElementSibling`, unfortunately. They aren't supported in PHP<8: https://3v4l.org/IRpHf

So this will need to use `nextSibling` and `previousSibling` instead.

Reply:
Good to know! Fixed.